apps/dashboard/lib/tasks/erb_render_helper.rb

```diff
@@ -0,0 +1,10 @@
+module ERBRenderHelper
+  # To add in common use cases in ERB rendering when needed
+  def groups
+    @groups ||= OodSupport::Process.groups.map(&:name)
+  end
+
+  def user_in_group(group)
+    groups.include?(group.to_s)
+  end
+end
```

Comment:
I don't think `lib/tasks` is the right place. Maybe `lib/erb`?  Also `user_in_group` should be `user_in_group?`.

Lastly I now wonder if using `OodSupport::User.new.groups` instead of `OodSupport::Process::Process.groups`, but that brings other questions of restructuring this to be a facade for `OodSupport::User` where we delegate all sorts of API. Maybe we should switch now, and revisit later. That'll mean we can launch with User-centric capabilities as a pose to Process-centric. And we can update it later keeping the same user facing functionality.

Reply:
Could also just put it in `app/models`. It is business logic. But these methods also seem misplaced. There is nothing ERB specific about turning `OodSupport::Process.groups.map(&:name)` and `OodSupport::Process.groups.map(&:name).include?(group.to_s)` into shorter methods.

```
irb(main):003:0> OodSupport::Process.groups.include?('PZS1008')
=> true
```

Also is `user_in_group(group)` actually a big improvement over `groups.include?(group)`? 

Where exactly do we want to use these shortcuts?

Reply:
The idea is to to have the module follow a "view helper pattern" where we can essentially use these helper methods inside of the erb files that are being rendered

Reply:
Which erb files

Reply:
Erb files that are being rendered in classes such as, announcement.rb or the motd formatter markdown_erb or plaintext_erb classes. What I'm basically doing is, I'm including this module in certain classes that render erb files that could potentially need to use these helper methods.

Reply:
> Which erb files

Really!? `form.yml.erb` comes to mind. Like all the PRs I'm doing now. We have an announcement in OOD _right now_ that computes groups. This is a very common pattern for a lot of our ERB files that we toggle some functionality/view based on group membership.